force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -147,7 +191,26 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is today or in the future, False otherwise.
      */
     public static Boolean isDateTodayOrFuture(Date dt) {
-        return null; // Replace null with the variable you used to store the result
+        //date that calls the method
+        Date firstDate = dt;
+        //second date
+        Date secondDate = Date.today();
+        //if date that calls the method (input) occurs after second date (today), e.g. is in the future, the return value is negative. return true.
+        Integer daysBetween =firstDate.daysBetween(secondDate);
+        if (daysBetween<0) {
+            return true;
+            
+        }
+        //if input date is 0 it is same day. return false. 
+        if (daysBetween==0) {
```

Comment:
Since the only other outcome is to return false this if statement isn't necessary